### ISSUE TITLE
Show return type annotation when show_signature_annotations = true

### ISSUE THREAD

**Issue by johnthagen** [mkdocstrings/python#4]
**Describe the solution you'd like**

When setting:

```yaml
plugins:
  - search
  - mkdocstrings:
      handlers:
        python:
          rendering:
            show_signature_annotations: true
````

It would be helpful if this also rendered the return type (e.g `-> int`)

<img width="751" alt="Screen Shot 2022-02-22 at 8 10 49 AM" src="https://user-images.githubusercontent.com/10340167/155139079-42da7b78-800c-46f5-b892-44ff00b21a6a.png">

This is the common, concise way that Python developers are used to reading types.

**Describe alternatives you've considered**

An extra configuration option, but I don't think that is necessary.

**Comment by pawamoy** [comment]
Fixed in version 0.6.4 :slightly_smiling_face:

**Comment by johnthagen** [comment]
Confirmed this is fixed in 0.6.4. Thanks!
